repository-data/webfiles/src/main/resources/site/src/scss/components/_navigation-tiles.scss

```diff
@@ -107,20 +107,20 @@
     line-height: 1.35;
     font-weight: bold;
     color: $blue;
+    margin-right: -4px;
+    margin-left: -4px;
 }
 
 .navigation-tile__link-text {
-    border-bottom: 1px solid;
+    text-decoration: underline;
```

Comment:
Just a note, there could be an accessibility reason for a border being used over an underline.